### PR TITLE
Validate contact form: Contact Form Validation

### PR DESCRIPTION
**Pull request by ClaudiaRojasSoto**
This Pull Request implements client-side validation for the contact form on the portfolio website. The implementation details and how it meets the specified requirements are described below.

_General Requirements_
Linting: It has been verified and ensured that there are no linter errors in the code.
GitHub Flow: The GitHub flow has been followed correctly, creating a new branch, making the changes in that branch, and finally creating this Pull Request for review.
Professional Documentation: Detailed and professional documentation has been provided, explaining the purpose and details of the changes made.

_HTML/CSS and JavaScript Requirements_

HTML and CSS: The recommended best practices for HTML and CSS have been followed, ensuring clean and structured code. A professional and consistent design has been used for the contact form, following the established design guidelines.
JavaScript: Client-side validation has been implemented for the contact form using JavaScript. The interactions and compliance with the requirements are detailed below:

_Lowercase Email Validation_
* When the user submits the form, it checks if the entered email is in lowercase.
* If the email is in lowercase, the form submission is allowed.
* If the email is not in lowercase, an error message is displayed to the user near the submit button, informing them of the error.
* The error message follows the established design guidelines and uses a style consistent with the overall website design.

_Additional Interactions_
* The recommended best practices for JavaScript, such as using const and let variables, and proper separation of logic into functions, have been followed.
* The code has been organized clearly and readably, using indentation and descriptive comments when necessary.
* Usability and user experience have been taken into account by providing clear visual feedback in case of validation errors.

This Pull Request meets the specified requirements for implementing client-side validation in the contact form. The code has been written according to best practices, and detailed and professional documentation has been provided.

### CODE REVIEW

File: js/index.js [hunk 1]
```diff
@@ -264,3 +349,22 @@ seeProjectsButton.forEach((button) => {
...
+  const emailValue = emailInput.value;
+  const lowercaseEmail = emailValue.toLowerCase();
+
+  if (emailValue !== lowercaseEmail) {
+    const errorElement = document.createElement('p');
+    errorElement.innerText = 'Send Error. Your email must be written in lowercase letters.';
+    errorElement.classList.add('error-message');
+    textArea.parentNode.insertBefore(errorElement, textArea.nextSibling);
+  }
+});
```
Comment (Zuhaib042) [inline js/index.js]: - [x] You validation is on point and its working but somehow the form is not getting submitted when I write email lowercase. I have checked it and got it working by writing following code after `if` condition
```
if (emailValue !== lowercaseEmail) {
    const errorElement = document.createElement('p');
    errorElement.innerText = 'Send Error. Your email must be written in lowercase letters.';
    errorElement.classList.add('error-message');
    textArea.parentNode.insertBefore(errorElement, textArea.nextSibling);
  } else {
    formValidator.submit();
  }
```
bug demonstration

https://github.com/ClaudiaRojasSoto/portfolio_1/assets/96683843/648ce8e9-e139-467c-97c1-c53eab08bd37

**Review by Zuhaib042 — CHANGES_REQUESTED** [review]
Hi @ClaudiaRojasSoto 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
✅ Professional Readme
✅ Descriptive PR

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: js/index.js (hunk 1 above)
Comment (ClaudiaRojasSoto) [reply js/index.js]: I forgot the else. Thank you.

**Review by Kossi-stack — APPROVED** [review]
## Status: Approved ✔️

**Hi @ClaudiaRojasSoto,**

You did a really great job! 💯 💯💯💯
Congratulations! 🎉 🎉 🎉 🎉 

128382030-38e7de91-f6f2-45cf-b54e-8068e0446e4c

You may close this Pull Request and merge the changes
Keep being awesome dev's and good luck with your next milestone!

## To Highlight

✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ README file is descriptive.
✅ Commit messages are meaningful.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding! 👏 👏 👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Pull request merged**